src/Helper/Names/DefaultNamesHelper.php

```diff
@@ -12,6 +13,16 @@ class DefaultNamesHelper implements NamesHelperInterface
 {
     use HelperTrait;
 
+    public function format(string $name): string
+    {
+        return sprintf(
+            '%s-%s-%s',
```

Comment:
Since now we may have an instance of a project in app - we should include project name to manifest name.
By default name should have format `[env]-[projectName]-[appName]-[manifestName]`